contrib/jenkins_tests/globals.sh

```diff
@@ -64,6 +64,18 @@ function do_github_status()
     "https://api.github.com/repos/$repo/statuses/${sha1}?access_token=$token"
 }
 
+function check_env()
+{
+	if [ ! $(git rev-parse --abbrev-ref HEAD | grep 'experimental_256B' >/dev/null 2>&1 || echo $?) ]; then
+		if [ $(ofed_info -s | grep 'MLNX_OFED_LINUX-3.2' >/dev/null 2>&1 || echo $?) ]; then
```

Comment:
is there additional indication in mofed headers that libvma will not compile?
maybe it is better to grep mofed header file for support instead of rely on specific version?